src/Aquifer.Jobs/Configuration/ConfigurationOptions.cs

```diff
@@ -3,12 +3,21 @@
 public class ConfigurationOptions
 {
     public bool IsDevelopment { get; init; }
+    public required string AquiferAdminBaseUri { get; init; }
     public required string AquiferApiBaseUri { get; init; }
     public required string KeyVaultUri { get; init; }
+    public required ConnectionStringOptions ConnectionStrings { get; init; }
```

Comment:
Connection strings are already present in the configuration; this allows accessing them via the same patterns as the API projects.